src/components/landingPage/LandingPageContainer.jsx

```diff
@@ -43,6 +44,10 @@ export class LandingPageContainer extends React.Component {
     const urlParams = new URLSearchParams(this.props.location.search);
     const login = urlParams.get('login');
     const register = urlParams.get('register');
+    // setUserId(this.props.userId);
```

Comment:
Please also take out this commented code if you no longer need them.

Reply:
Thanks for this feedback, I have removed it.